docs/src/pages/reference/configuration/output.md

```diff
@@ -726,6 +726,20 @@ Type: `Object`.
 
 Exactly the same as the `override.operations` but this time you can do it by <a href="https://swagger.io/docs/specification/grouping-operations-with-tags/" target="_blank">tags</a>
 
+#### operationId
+
+Type: `Function`.
+
+```ts
+// type signature
+(operation: OperationObject, route: string, verb: Verbs) => string;
+```
+
+Function to override the generate operation id. The id is then used to generate all types and functions names.
+You can see <a href="https://github.com/anymaniax/orval/blob/next/src/core/getters/operation.ts" target="_blank">here</a> how we compute that id by default (`getOperationId`).
+
+__Important notice:__ The id returned here should always be unique by route and verb otherwise you'll get naming conflicts when generating types/functions.
```

Comment:
Can you add a configuration example showing how you would use this?    I think that would help users understand for example what you would do.

Reply:
Sure, I added with an actual example 👍